HenBot/TagsQueryBuilder.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace HenBot
+{
+    public class TagsQueryBuilder
+    {
+        static readonly string _alwaysBannedTags = "+-animated";
```

Comment:
If you're using RestSharp build-in .AddParameter method, you don't have to you escaped characters in the query parameters

```suggestion
        static readonly string _alwaysBannedTags = " -animated";
```